WinUIGallery/ControlPages/DesignGuidance/TypographyPage.xaml

```diff
@@ -39,8 +39,8 @@
         </Grid.RowDefinitions>
         <RichTextBlock>
             <Paragraph>
-                Type helps provide structure and hierarchy to UI. The default font for Windows is
-                <Hyperlink NavigateUri="https://learn.microsoft.com/windows/apps/design/downloads/#fonts">Segoe UI Variable</Hyperlink>.
+                Type helps provide structure and hierarchy to UI. The default font for Windows is <Hyperlink NavigateUri="https://learn.microsoft.com/windows/apps/design/downloads/#fonts">Segoe UI Variable</Hyperlink>
+                .
```

Comment:
The line break before the period will be converted to a space . Visual Studio keeps doing this on every "format document" . :⁠-⁠(